### DISCUSSION TITLE
Cannot search for issues related to layers

### DISCUSSION THREAD

**Discussion by geoffreyvanwyk** [syl20bnr/spacemacs#16239] [General]
When I try to search for an issue related to a layer, the results contain many issues that are not related. This is due to issue authors many times pasting the list of all their installed layers. That is good for finding a solution, but not for searching.

**Comment by alexey0308** [comment]
Some of the issues are marked with the [labels](https://github.com/syl20bnr/spacemacs/labels).

**Reply by geoffreyvanwyk** [reply]
That is the answer, and it will depend on the collaborators labelling as many as necessary and possible.
